WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -112,6 +113,9 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main_activity);
 
+        // Allow vector drawables on pre-Lollipop (i.e. < API 21).
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
```

Comment:
Sounds good.  I'll cherry pick your commit and remove it from `WPMainActivity.OnCreate`.